Doc/Makefile

```diff
@@ -107,7 +109,7 @@ clean:
 
 venv:
 	$(PYTHON) -m venv venv
-	./venv/bin/python3 -m pip install -U Sphinx
+	./venv/bin/python3 -m pip install -U Sphinx blurb
```

Comment:
As above, suggest setting a version requirement here:
`./venv/bin/python3 -m pip install -U Sphinx 'blurb>=1.0.2dev1'`
That should be future-proof.